index.js

```diff
@@ -21,18 +21,27 @@ const getGlobal = property => {
 	}
 };
 
-const document = getGlobal('document');
-const Headers = getGlobal('Headers');
-const Response = getGlobal('Response');
-const ReadableStream = getGlobal('ReadableStream');
-const fetch = getGlobal('fetch');
-const AbortController = getGlobal('AbortController');
-const FormData = getGlobal('FormData');
+const globals = [
```

Comment:
Use [`Proxy`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy).

Reply:
Really? Proxy is not available on all platforms and can not be mocked or transpiled.

Reply:
Personally, I would rather call `getGlobal()` inline where we need it, which will have the same effect. It's a little more verbose, but is much simpler to  read and understand.

Reply:
@luxferresum

![image](https://user-images.githubusercontent.com/36894700/61814898-601efd00-ae49-11e9-95d1-ad899534b085.png)

Really? You're worried about IE?

@sholladay Yup, that would do the same but IMHO `Proxy` is the way to go. Code is much readable. `globals.URL` > `getGlobal('URL')`

Reply:
Yes!
Many people still need to support IE!

And I don't think it's much harm to explicit list the globals.